src/main/scala/uk/gov/nationalarchives/notifications/messages/EventMessages.scala

```diff
@@ -286,27 +284,36 @@ object EventMessages {
             ),
             reference = s"${metadataReviewRequestEvent.consignmentReference}"
           )
+        ),
+        Some(
+          GovUKEmailDetails(
+            templateId = eventConfig("gov_uk_notify.metadata_review_requested_tb_template_id"),
+            userEmail = metadataReviewRequestEvent.userEmail,
+            personalisation = Map(
+              "consignmentReference" -> metadataReviewRequestEvent.consignmentReference,
+            ),
+            reference = s"${metadataReviewRequestEvent.consignmentReference}"
+          )
         )
-      } else None
+      )
     }
   }
 
   implicit val metadataReviewSubmittedEventMessages: Messages[MetadataReviewSubmittedEvent, Unit] = new Messages[MetadataReviewSubmittedEvent, Unit] {
     override def context(event: MetadataReviewSubmittedEvent): IO[Unit] = IO.unit
-    override def govUkNotifyEmail(metadataReviewSubmittedEvent: MetadataReviewSubmittedEvent, context: Unit): Option[GovUKEmailDetails] = {
-      if (eventConfig("gov_uk_notify.on").toBoolean) {
-        Some(
-          GovUKEmailDetails(
-            templateId = eventConfig("gov_uk_notify.metadata_review_submitted_template_id"),
-            userEmail = config.getString("tdr_inbox_email_address"),
-            personalisation = Map(
-              "consignmentReference" -> metadataReviewSubmittedEvent.consignmentReference,
-              "urlLink" -> metadataReviewSubmittedEvent.urlLink,
-            ),
-            reference = s"${metadataReviewSubmittedEvent.consignmentReference}"
-          )
+    override def govUkNotifyEmail(metadataReviewSubmittedEvent: MetadataReviewSubmittedEvent, context: Unit): List[Option[GovUKEmailDetails]] = {
+      val templateId = if (metadataReviewSubmittedEvent.status == "Completed") eventConfig("gov_uk_notify.metadata_review_approved_template_id") else eventConfig("gov_uk_notify.metadata_review_rejected_template_id")
+      List(Some(
```

Comment:
Would like it if we only return this message if `gov.external_emails_on` is true